public/static/locales/de/editProfile.json

```diff
@@ -23,6 +23,7 @@
   "cancel": "Abbrechen",
   "termAndCondition": "Ich stimme den <0>Nutzungsbedingungen</0> der Plant-for-the-Planet Plattform zu.",
   "termAndConditionError": "Bitte stimme den Nutzungsbedingungen zu",
+  "member": "Mitglied seit {{date}}",
```

Comment:
In public/static/locales/en/editProfile.json this new resource is named `memberSince`, so this is probably a bug.

Reply:
I fixed this myself by adding German translations for better testing.